pages/common/git-verify-pack.md

```diff
@@ -0,0 +1,16 @@
+# git verify-pack
+
+> Validate packed Git archive files.
```

Comment:
```suggestion
> Verify packed Git archive files.
```

I think we should use consistent wording, especially since verify and validate are not synonyms.